plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -554,6 +567,11 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
             artifactHandler.showTransformationSummary(jobId)
         }
 
+    private fun displayFeedbackNotificationAction() =
+        NotificationAction.createSimple(message("codemodernizer.notification.warn.submit_feedback")) {
+            FeedbackDialog(project, productName = "Amazon Q").showAndGet()
```

Comment:
Is there a Constant we can use for `Amazon Q` that we can use instead?

Reply:
Nice to have: Make this a constant to be shared for cleaner code. Something like `AMAZON_Q_FEEDBACK_DIALOG_KEY`